src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write code here
+const peopleList = document.body.children[1].children;
```

Comment:
Better use querySelector (or querySelectorAll) instead to make your code cleaner
https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector